src/test/scala/strawman/collection/test/Test.scala

```diff
@@ -288,6 +288,15 @@ class StrawmanTest {
     println(xs17.to(List))
   }
 
+  def mapOps(xs: strawman.collection.Map[Int, String]): Unit = {
+    // val xs1 = xs.map { case (k, v) => (v, k) }
+    // val xs2: strawman.collection.Map[String, Int] = xs1
+    val xs1 = xs.map(kv => (kv._2, kv._1))
+    val xs2: strawman.collection.Iterable[(String, Int)] = xs1
```

Comment:
As you can see in the above lines:
- Using the `.map { case (k, v) => …` syntax breaks type inference ;
- The inferred type of `xs1` is `Iterable[(String, Int)]` instead of `Map[String, Int]`.